src/test/regress/sql/multi_colocation_utils.sql

```diff
@@ -7,7 +7,7 @@ ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1300000;
 -- ===================================================================
 
 CREATE SEQUENCE colocation_test_seq
-    MINVALUE 1
+    MINVALUE 1000
```

Comment:
Renaming makes it harder to review and rebase, but we can change it just before the merge.
I think multi_colocation_utils and multi_colocation are also good options.